submissions/KostiantynOkhorzin/dom/script.js

```diff
@@ -98,23 +100,23 @@ document.addEventListener('DOMContentLoaded', () => {
     ];
 
     const listBtns = document.querySelector('.side-menu');
-    const content = document.querySelector('.content');
-    const burger = document.querySelector('.burger');
-
+    const mainContent = document.querySelector('.content');
+    const btnBurger = document.querySelector('.burger');
+    const body = document.body;
 
     const renderBtns = (arr, list) => {
-        arr.map((item, i) => {
+        for (let i = 0; i < arr.length; i++) {
             list.insertAdjacentHTML('beforeend', `
                 <li class="side-menu__item">
-                    <button class="side-menu__btn" data-season='${i}'>Season ${i + 1}</button>
+                    <button class="side-menu__btn" data-season='${i}'>Season ${i+1}</button>
                 </li>
             `);
-        })
+        }
     };
 
     const renderContent = (i = 0, arr = data) => {
         const {title, descr, img} = arr[i];
```

Comment:
If someone change iteration of seasons then on event click we get incorect season.

Reply:
I do not understand who can change the iteration of the seasons?
And what should I do?